docs/source/examples/autosweep.py

```diff
@@ -0,0 +1,35 @@
+"Show autosweep_1d functionality"
+import numpy as np
+import gpkit
+from gpkit import units, Variable, Model
+from gpkit.tools.autosweep import autosweep_1d
+from gpkit.small_scripts import mag
+
+A = Variable("A", "m**2")
+l = Variable("l", "m")
+
+m1 = Model(A**2, [A >= l**2 + units.m**2])
+tol1 = 1e-3
+bst1 = autosweep_1d(m1, tol1, l, [1, 10])
```

Comment:
should we use `m1.autosweep`, as in the text of the docs? I assume that returns the same thing?